test/fakeintake/server/server_test.go

```diff
@@ -283,27 +328,28 @@ func TestServer(t *testing.T) {
 			},
 		})
 
-		clock.Add(10 * time.Minute)
+		// Retention is 5 minutes
+		clock.Add(2 * time.Minute)
 
-		response10Min, err := http.Get(fi.URL() + "/fakeintake/payloads?endpoint=/totoro")
+		response2Min, err := http.Get(fi.URL() + "/fakeintake/payloads?endpoint=/totoro")
 		require.NoError(t, err, "Error on GET request")
-		defer response10Min.Body.Close()
+		defer response2Min.Body.Close()
 
-		var getResponse10Min api.APIFakeIntakePayloadsRawGETResponse
-		json.NewDecoder(response10Min.Body).Decode(&getResponse10Min)
-		assert.Len(t, getResponse10Min.Payloads, 2, "should contain two elements before cleanup %+v", getResponse10Min)
+		var getResponse2Min api.APIFakeIntakePayloadsRawGETResponse
+		json.NewDecoder(response2Min.Body).Decode(&getResponse2Min)
+		assert.Len(t, getResponse2Min.Payloads, 2, "should contain two elements before cleanup %+v", getResponse2Min)
 
-		clock.Add(10 * time.Minute)
+		clock.Add(7 * time.Minute)
 
-		response20Min, err := http.Get(fi.URL() + "/fakeintake/payloads?endpoint=/totoro")
+		response7Min, err := http.Get(fi.URL() + "/fakeintake/payloads?endpoint=/totoro")
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
This is retention at `7+2` = `9` minutes